index.html

```diff
@@ -955,172 +961,325 @@ <h2>Web Publication Lifecycle</h2>
+                    </li>
+                    <li>
+                        Return <var>manifest</var>.
+                    </li>
+                </ol>
+            </section>
+            
+            <section id="processing-reading-order">
+                <h3>Processing the default reading order</h3>
+                <p>
+                    The <dfn data-lt="processing the default reading order">steps for processing the default reading order</dfn> are given by the following algorithm.
+                </p>
+                <ol>
+                    <li>
+                        If <var>manifest</var>["reading_order"] is not empty, return it and terminate this algorithm.
+                    </li>
+                    <li>
+                        If <var>manifest</var>["resources"] is not empty, let <var>navigation document link</var> be the first <a>PublicationLink</a> whose <code>rel</code> member contains the token <code>contents</code>.
+                    </li>
+                    <li>
+                        If <var>navigation document link</var> is not <code>null</code> and its <code>href</code> member is not <code>empty</code>, let <var>navigation document URL</var> be the result of parsing the value of the <code>href</code> attribute, relative to the <var>manifest URL</var>.
+                    </li>
+                    <li>
+                        If <var>navigation document URL</var> is empty, set <var>navigation document URL</var> to <var>document URL</var>.
+                    </li>
+                    <li>
+                        Let <var>request</var> be a new [[!FETCH]] request, whose URL is <var>navigation document URL</var>, and whose context is "<code>publication</code>".
+                    </li>
+                    <li>
+                        Await the result of performing a [[!FETCH]] with <var>request</var>, letting <var>response</var> be the result.
+                    </li>
+                    <li>
+                        If <var>response</var> is a network error, terminate this algorithm.
+                    </li>
+                    <li>
+                        Let <var>document</var> be the result of UTF-8 decoding <var>response</var>'s <a data-cite="!FETCH#concept-request-body">body</a>.
+                    </li>
+                    <li>
+                        If <var>document</var> is not a valid [[!HTML]] document, terminate this algorithm.
+                    </li>
+                    <li>
+                        Let <var>default reading order</var> be the result of processing <var>text</var>:
+                        <ol>
+                            <li>
+                                If <var>document</var> contains a <code>nav</code> element:
+                                <ol>
+                                    <li>Extract a list of resource paths referenced from the <code>href</code> attribute of all <code>a</code> elements.</li>
```

Comment:
agree with the logic, but at some point we’ll need to define it a bit more accurately (e.g. "resource paths" sounds ambiguous wrt to the URL standard, probably add "in document order", etc).

Reply:
For now, I've mostly re-used what we had in the infoset for the default reading order.

I really dislike our current status quo, and would rather drop the ability of defining the default reading order in an HTML document than continue to make this algorithm even more complex than it is.